docs/configuration.md

```diff
@@ -137,6 +137,13 @@ statistics:
 # Export the metric with the original CloudWatch timestamp (General Setting for all metrics in this job)
 [ addCloudwatchTimestamp: <boolean> ]
 
+# Enables the inclusion of past metric data points from the CloudWatch response if available.
+# This is useful when a metric is configured with a 60-second period and a 300-second duration, ensuring that all
+# five data points are exposed at the metrics endpoint instead of only the latest one.
+# Note: This option requires `addCloudwatchTimestamp` to be enabled
+# (General Setting for all metrics in this job)
+[ addHistoricalMetrics: <boolean> ]
```

Comment:
WDYT about calling it `exportAllDataPoints`? I worry with a name like `addHistoricalMetrics` it the feature could be confused for a historical data extract.